### PR TITLE
Add "Feed" Component

### PR DESCRIPTION
**Pull request by hankerve**
- Import more colors from Zeplin (I exported colors from Zeplin and some of the naming might be changed. I will change it back if it breaks your code🤕🙇‍♂️).
- Changed Work-Sans Font file since iOS can't determine exact font sizing from current file.
- Added extension to "Font" for use in SwiftUI.
- Added "Feed" component.
- Added "TagSelected" component.

### CODE REVIEW

File: FightPandemics/Extensions/Font+Additions.swift [hunk 1]
```diff
@@ -0,0 +1,41 @@
+//
+//  Font+Additions.swift
+//  FightPandemic [IOS]
+//
+//  Generated on Zeplin. (9/17/2020).
```
Comment (stavares843) [inline FightPandemics/Extensions/Font+Additions.swift:5]: do we need the Generated on Zeplin comment?

Comment (hankerve) [reply FightPandemics/Extensions/Font+Additions.swift:5]: Thanks! Will remove.

**Review by Power186 — COMMENTED** [review]
Dark Mode?

**Comment by hankerve** [comment]
> Dark Mode?

Hi Scott! Currently we don't have a design for dark mode in Zeplin. What I can do is to change the color of the text to white in dark mode but I have not idea for how other colors should change. We sure need to reflect this in the product meeting and ask for a separate set of design.

**Comment by Power186** [comment]
> > Dark Mode?
> 
> Hi Scott! Currently we don't have a design for dark mode in Zeplin. What I can do is to change the color of the text to white in dark mode but I have not idea for how other colors should change. We sure need to reflect this in the product meeting and ask for a separate set of design.

No design for dark mode needed.  No separate design needed.  You set the environment for light mode in ContentView and whether a user has light mode or dark mode enabled won't matter, it will show the color/design we made in the app regardless.  I'll update the change and I'd be happy to show you how to do this come next meeting if you'd like.  Other than that, nice work!

**Comment by hankerve** [comment]
> > > Dark Mode?
> > 
> > 
> > Hi Scott! Currently we don't have a design for dark mode in Zeplin. What I can do is to change the color of the text to white in dark mode but I have not idea for how other colors should change. We sure need to reflect this in the product meeting and ask for a separate set of design.
> 
> No design for dark mode needed. No separate design needed. You set the environment for light mode in ContentView and whether a user has light mode or dark mode enabled won't matter, it will show the color/design we made in the app regardless. I'll update the change and I'd be happy to show you how to do this come next meeting if you'd like. Other than that, nice work!

Thanks Scott! The current UI looks fine on my phone under dark mode. But some of the icons probably need to be transparent and the color scheme might need to change (under dark mode). Maybe we will consider this issue in the future when everything's done!
IMG_1570

**Pull request closed without merging**